.github/workflows/test.yml

```
@@ -2,7 +2,7 @@ name: test
 
 on:
   push: # Run on pushes to the default branch
-    branches: [main]
+    branches: [main,upgrade-gh-action-artifact-upload-download]
```

```suggestion
    branches: [main]
```
Remove current branch added for testing CI before merging the PR